lib/jackal/callback.rb

```diff
@@ -108,7 +108,8 @@ def forward(payload, dest=nil)
       source = Carnivore::Supervisor.supervisor[dest]
       if(source)
         info "Forwarding payload to output destination... (#{source})"
-        debug "Forwarded payload: #{payload.inspect}"
+        output = Carnivore::Config.get(:pretty_print_payload) ? payload.pretty_inspect : payload.inspect
```

Comment:
Jackal should never reference the top level config (except when it should, which is generally never). Using the `#app_config` would be preferred here:

```ruby
app_config(:pretty_print_payload)
```

However, debug is just for debug, so I don't think having a switch for this really matters. We can just force it to always be formatted and if it's too much output, just bump up to :info 

```ruby
debug "Forwarded payload: #{output.pretty_inspect}"
```